src/reducers/app/reducers.ts

```diff
@@ -134,9 +135,12 @@ export const app = (state = initialState, action) => {
   switch (action.type) {
     case SET_FILTER:
       return { ...state, filters: action.filters };
-    case SET_ITEMS:
+    case SET_ITEMS: {
       const items = { ...state.items, [action.entity]: action.items };
       return { ...state, items };
+    }
+    case INIT_ITEMS:
```

Comment:
Not just for INIT_ITEMS, but in general, having TSDoc comments on the functions that get triggered by these events would be helpful with debugging